test/Microsoft.SqlTools.ServiceLayer.Test.Common/TestData/ObjectExplorer/Baselines/AllSqlObjects.txt

```diff
@@ -42,6 +42,7 @@ NodeType: Constraint Label: CK_Employee_MaritalStatus SubType: Status:
 NodeType: Constraint Label: CK_Employee_SickLeaveHours SubType: Status:
 NodeType: Constraint Label: CK_Employee_VacationHours SubType: Status:
 NodeType: Index Label: NonClusteredIndex-Login (Non-Unique, Non-Clustered) SubType: Status:
+NodeType: Index Label: PK_Employee_BusinessEntityID (Unique, Clustered) SubType:PrimaryKey Status:
```

Comment:
Would be great if the test could cover an FK too.

Reply:
there's already an example for FK (FK_Employee_Person_BusinessEntityID) in SSMS foreign keys show up under keys only not indexes  

---
In reply to: [154440262](https://github.com/Microsoft/sqltoolsservice/pull/557#discussion_r154440262) [](ancestors = 154440262)